Looks like we may have an issue with the docs build after reintroducing `torchvision` build:
```
Oct 26 23:30:05 build finished with problems, 32 warnings.
Oct 26 23:30:05 /var/lib/jenkins/workspace/docs/src/pytorch-sphinx-theme/pytorch_sphinx_theme/search.html:21: RemovedInSphinx30Warning: To modify script_files in the theme is deprecated. Please insert a <script> tag directly in your theme instead.
Oct 26 23:30:05   {% trans %}Please activate JavaScript to enable the search
Oct 26 23:30:05 Makefile:38: recipe for target 'html' failed
```

--- a/docs/Makefile
+++ b/docs/Makefile
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    ?= -j auto -WT --keep-going
+SPHINXOPTS    ?= -j auto
 SPHINXBUILD   ?= sphinx-build
 SPHINXPROJ    ?= PyTorch
 SOURCEDIR     ?= source

